javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,9 @@
+function ExampleConstructor() {}
+
+console.log('value of ExampleConstructor.prototype: ', ExampleConstructor.prototype);
+console.log('type of ExampleConstructor.prototype: ', ExampleConstructor.prototype);
```

Comment:
The second one should be its type, not its value

Reply:
Noted.

Reply:
![javascript-contructors-example-constructors-2](https://user-images.githubusercontent.com/90303824/168722210-6f655dea-a1f1-4f20-b3fb-0b817158cc9e.png)